src/atoms/Tooltip/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import styles from './tooltip.module.scss';
+import classnames from 'classnames';
+
+function Tooltip ( props ) {
+  const {
+    children,
+    onClick,
+    className
+  } = props;
+
+  return (
+    <div className={ classnames(className, styles['tooltip']) }>
+      <Icon icon='tooltip' 
+        onClick={ onClick }
```

Comment:
`Icon` doesn't take an `onClick` prop - is it needed anywhere else in the component?